src/context.ts

```diff
@@ -492,6 +492,9 @@ class Context extends EventEmitter implements ContextEventEmitter {
   }
 
   _urlForPath(path: string, isWebSocket: boolean, options: MojoURLOptions): string {
+    if (path === '') {
+      path = '/';
+    }
     path = this.req.basePath + path;
```

Comment:
Why not just `path = this.req.basePath + (path === '' ? '/' : path);` like you started out with? Was there something else wrong?

Reply:
Right, better this way. Done and squashed